os/dutils.sh

```diff
@@ -57,6 +60,64 @@ function FIND_DBGBINFILE()
 	KERNEL_DBG_BIN="tinyara$EXEEXT"
 }
 
+function TRAP_MENU()
+{
+	echo ======================================================
+	echo "  \"Run TRAP tool for\""
+	echo ======================================================
+	echo "  \"1. Default binary\""
+	echo "  \"2. Custom binary\""
+	echo "  \"x. Exit\""
+	echo ======================================================
+
+	read SELECT_BIN
+	case ${SELECT_BIN,,} in
+		1|default)
+			if [ -z "$(ls -A $BINDIR)" ]; then
+				echo "No output file, Build the code and run TRAP again."
+				exit 1
+			fi
+			echo "Enter the name of the crash log file:"
+			read LOG
+			cp $LOG $TRAPDIR
```

Comment:
I dont understand why we are doing this copy? Will it not overwrite the user entry?

Reply:
We are just copying the logs from custom to trap directory which are being used in ramdumpParser, so no overwriting will be there

Reply:
If there is the same file, is it ok?

Reply:
No need to check `LOG` existence?